docs/getstarted/getting-started.md

```diff
@@ -0,0 +1,277 @@
+---
+Order: 1
+Area: getstarted
+TOCTitle: Tutorial
+ContentId: 72ad9b70-5227-4032-81d7-6aec00a1e8f8
+PageTitle: Getting started with Visual Studio Code
+DateApproved: 08/01/2024
+MetaDescription: This tutorial gives you an overview of the key features of Visual Studio Code to help you get started quickly.
+---
+# Tutorial: Get started with Visual Studio Code
+
+In this tutorial, you learn about the key features of Visual Studio Code to help you get started with coding quickly. You learn about the different components of the user interface and how to customize it to your liking. You then write some code and use the built-in code editing features, such as IntelliSense and Code Actions, and you learn about running and debugging your code. By installing a language extension, you add support for a different programming language.
+
+<iframe src="https://www.youtube.com/embed/B-s71n0dHUk?autoplay=true" width="640" height="320" allowFullScreen="true" frameBorder="0" title="Getting Started with Visual Studio Code"></iframe>
+
+## Prerequisites
+
+- [Download Visual Studio Code](https://code.visualstudio.com/download)
+- [Set up Visual Studio Code on your computer](/docs/setup/setup-overview.md)
+- [Install Node.js](https://nodejs.org/)
+
+## Step 1: Open a folder in VS Code
+
+VS Code is a great lightweight editor where you can with individual files seamlessly, for example for quick edits or to view a file. However, by opening a folder, also known as a *workspace*, you unlock more features, such as folder-specific configuration settings, restoring the UI state for that folder, debug configurations, and more. Get more info about [workspaces](/docs/editor/workspaces.md).
+
+Let's start by creating a folder and opening it in VS Code. You'll use this folder throughout the tutorial.
+
+1. Create a new folder `vscode101` on your computer.
+
+1. Open Visual Studio Code.
+
+    When you first open VS Code, you should see the **Welcome** page with different actions to get started. You can also explore some of the VS Code product walkthroughs.
+
+1. On the Welcome page, select **Open Folder...**, and then select the folder you created.
+
+    The VS Code window reloads and you should see the folder name at the top of the **Explorer** view. You'll use the Explorer view to view and manage the files and folders in your workspace.
+
+    ![Screenshot that shows VS Code after opening a folder, highlighting the Explorer view.](images/getting-started/open-folder.png)
+
+    Alternatively, you can select **Open Folder...** from the **File** menu or press `kb(workbench.action.files.openFolder)` to open a folder.
+
+> **Tip**: When you use the command line, pass the folder name as a parameter to directly launch VS Code for that folder. For example, enter `code .` to open the current folder in VS Code.
```

Comment:
nit: "When" sounds like users will be using the CLI at some point, but this wouldn't be a requirement right?

```suggestion
> **Tip**: If you use the command line to work with VS Code, pass the folder name as a parameter to directly launch VS Code for that folder. For example, enter `code .` to open the current folder in VS Code.
```